src/dateAssesor.js

```diff
@@ -0,0 +1,18 @@
+const settings = require('./appSettings');
+
+const dateAssesor = (dateToConvert) => {
+	const facebookDates = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'];
+
+	const month = facebookDates.indexOf(dateToConvert.substring(0,3));
+	const day = dateToConvert.substring(3);
+	const date = new Date(new Date().getFullYear(), month, day);
+
+	const isPastEvent = new Date() > date; // check if date is in the past
+	const isSoonEvent = new Date().getMonth() + 3 - date.getMonth(); // check if event occurs in less than 3 month
+
+	const result = isSoonEvent >= 0 && isSoonEvent <= settings.maxMonthInFuture && !isPastEvent ? true : false;
+
+	return result;
```

Comment:
Why is there a `+ 3` here? Since below you check for `settings.maxMonthInFuture` you probably just want the delta between now and the event. I might be missing the point, however. Also, this is a matter of preference, but you could save a variable assignment and return the check directly.

```suggestion
	const monthDelta = new Date().getMonth() - date.getMonth();

	return monthDelta >= 0 && monthDelta <= settings.maxMonthInFuture && !isPastEvent;
```